src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -218,13 +270,19 @@ class StacktraceLink extends AsyncComponent<Props, State> {
   }
   renderBody() {
     const {config, sourceUrl} = this.match || {};
+    const {isDismissed} = this.state;
+
     if (config && sourceUrl) {
       return this.renderMatchWithUrl(config, sourceUrl);
     }
     if (config) {
       return this.renderMatchNoUrl();
     }
 
+    if (isDismissed || isDismissed === null) {
```

Comment:
@MeredithAnya would it be too much to add a new state variable called `promptIsLoaded` and make sure that is `true` before we check `isDismissed`?

Reply:
haha nah I can do that, I was really just looking to how this was done in other places, but I agree probs better to be explicit here